create_aio_app/template/{{cookiecutter.project_name}}/conftest.py

```diff
@@ -138,11 +137,20 @@ def tables(db):
     '''
 
     return await aiopg.sa.create_engine(**test_config['postgres'])
+
+
+@pytest.fixture
+async def postgres_engine(loop):
+    {%- if cookiecutter.use_postgres == 'aiopg' %}
+    import asyncpg
+    return await asyncpg.connect(**test_config['postgres'])
+    {%- endif %}
+
```

Comment:
I think that for compatibiliti with tests u should use asyncpgsa and just rewrite sa_engine some like that:

```python
from sqlalchemy import create_engine

...

return await create_engine(dsn, echo=True, module=asyncpgsa)
```

what do u think?